plugins/rbac/src/components/CreateRole/PermissionPoliciesFormRow.tsx

```diff
@@ -149,39 +149,37 @@ export const PermissionPoliciesFormRow = ({
             )}
           />
           <div style={{ width: '23%', alignSelf: 'center', flexGrow: 1 }}>
-            {permissionPoliciesRowData.isResourced &&
-              !!conditionRulesData?.[`${permissionPoliciesRowData.plugin}`]?.[
-                `${permissionPoliciesRowData.permission}`
-              ]?.rules.length && (
-                <IconButton
-                  title=""
-                  color="primary"
-                  hidden={
-                    !permissionPoliciesData?.pluginsPermissions[
-                      permissionPoliciesRowData.plugin
-                    ]?.policies[permissionPoliciesRowData.permission]
-                      ?.isResourced
-                  }
-                  aria-label="configure-access"
-                  className={classes.conditionalAccessButton}
-                  onClick={() => setSidebarOpen(true)}
-                  disabled={!!conditionRulesError}
+            {!!conditionRulesData?.[`${permissionPoliciesRowData.plugin}`]?.[
+              `${permissionPoliciesRowData.permission}`
+            ]?.rules.length && (
+              <IconButton
+                title=""
+                color="primary"
+                hidden={
+                  !permissionPoliciesData?.pluginsPermissions[
+                    permissionPoliciesRowData.plugin
+                  ]?.policies[permissionPoliciesRowData.permission]?.isResourced
```

Comment:
Once above comment is resolved you can simple use this `permissionPoliciesRowData.isResourced`. And better to add this above as firstly the resource type needs to be resourced then only it might have rules.